### PR TITLE
added some important functions

### PR DESCRIPTION
**Pull request by arpit1997**
Referencing the issue #37

### CODE REVIEW

**Comment by RNabel** [comment]
Hey! I'm currently swamped with work so will have a look at the PRs next week.

Keep up the great work :)

**Comment by arpit1997** [comment]
Thank you for reply.  I am happy to contribute here. Review my pull request and tell me if any mistakes are there
Thank you

File: docs/quickstart.rst [hunk 1]
```diff
@@ -67,6 +67,66 @@ Getting list of files
...
+        folder_selected=inp
+        for element in structure:
+            if type(element) is dict:
+                if element["title"]==folder_selected:
+                    struc=element["list"]
+                    browsed.append(folder_selected)
+                    print("Inside "+folder_selected)
+                    return folder_browser(struc,element['id'])
+                    
+ here structure is the list of folders that is present
```
Comment (RNabel) [inline docs/quickstart.rst]: This comment is not needed as the parameter's new name indicates what it contains.

**Comment by arpit1997** [comment]
Thank you for your suggestions in PR. I did not paid much attention to it and sent a PR without checking much. Sorry about that.. I guess i modified testutil.py by mistake so i will remove them. I will send a new PR soon.

File: docs/quickstart.rst [hunk 2]
```diff
@@ -67,6 +67,66 @@ Getting list of files
...
+            if type(element) is dict:
+                if element["title"]==folder_selected:
+                    struc=element["list"]
+                    browsed.append(folder_selected)
+                    print("Inside "+folder_selected)
+                    return folder_browser(struc,element['id'])
+                    
+ here structure is the list of folders that is present 
 
 You will see title and id of all the files and folders in root folder of your Google Drive. For more details, take a look at documentation: `File listing made easy`_
```
Comment (arpit1997) [inline docs/quickstart.rst]: I guess there is no typo in the documentation.

**Comment by arpit1997** [comment]
Please review the PR again. I have fixed the issues related to linting, formatting and naming conventions. Let me know if there is anything wrong.

**Comment by grant** [comment]
Sorry this PR was hanging for so long. The former reviewer left Google.
This PR looks good. Thanks for the contribution!

**Pull request merged**